### ISSUE TITLE
Conflict report doesn't reflect changes (unmapped OTUs)

### ISSUE THREAD

**Issue by bredelings** [OpenTreeOfLife/opentree#929]
I unmapped some OTUs for pg_2542.
However, the conflict report still shows conflict information as if those OTUs were mapped.
https://tree.opentreeoflife.org/curator/study/edit/pg_2542/?tab=trees&tree=tree5472&conflict=ott

**Comment by bredelings** [comment]
![conflict-bug](https://cloud.githubusercontent.com/assets/140268/13265265/045b9cde-da40-11e5-87d2-dd85064a372d.png)

**Comment by jar398** [comment]
Fixed version never got deployed to api.opentreeoflife.org.
(Here's what I did to diagnose:
`ssh api tail repo/reference-taxonomy/service/nohup.out`
  - this proved that the cache was being used
`ssh api grep reference-taxonomy log/messages | tail`
  - this proved that there was no update after the initial commit on Feb 17
open https://github.com/OpenTreeOfLife/reference-taxonomy/pull/191
  - says that the PR that fixed it was "five days ago" which I guess was
Feb 18)
My forgetfulness.
I will update now. The service will be down for a minute or two while it
reloads.
